source/vmware/vmware-overview.rst

```diff
@@ -0,0 +1,380 @@
+.. _minio-vmware:
+
+=======================
+MinIO on VMware vSphere
+=======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+Overview
+--------
+
+MinIO is a Kubernetes-native, high performance object storage suite that
+maintains AWS S3 API compatibility, allowing applications to seamlessly migrate
+to private or hybrid cloud infrastructures. VMware Cloud Foundation (VCF) with
+Tanzu is an enterprise-grade platform for running both traditional and modern
+Kubernetes-based applications in production with all the power of virtualized
+storage, networking, and associated management. 
```

Comment:
Customers visiting this place would have already decide how their VMware on prem looks like and if we include this then our docs will have to be in sync with the various ways VMware can choose productize their vSAN DPP offering
```suggestion
MinIO is a Kubernetes-native, high performance object storage suite that
maintains AWS S3 API compatibility, allowing applications to seamlessly migrate
to private or hybrid cloud infrastructures.  
```